content/docs/reference/organizing-stacks-projects.md

```diff
@@ -96,22 +96,65 @@ and testing. In that case, we'll have six distinct stacks, that pair up together
 The way Pulumi programs communicate information for external consumption is by using stack exports. For instance,
```

Comment:
```suggestion
The way Pulumi programs communicate information for external consumption is by using stack exports. For example,
```

Reply:
Note that many of the changes you are proposing here are not in text that is part of this PR.  Editorial pass over the full docs definitely appreciated - but might be easiest (and also help with consistency) to just do it on the raw doc content instead of on the bits and pieces you happen to see in PRs :-).